framework/wasm-adapter/src/api/managed_types/static_var_api_node.rs

```diff
@@ -81,19 +81,15 @@ impl StaticVarApiImpl for VmApiImpl {
         unsafe { CALL_VALUE_MULTI_ESDT_HANDLE }
     }
 
-    fn set_scaling_factor_cached(&self, decimals: usize) -> i32 {
-        let handle = const_handles::get_scaling_factor_handle(decimals);
-        let value: i64 = 10i64.pow(decimals as u32);
-        self.bi_set_int64(handle, value);
-        self.set_initialized(decimals);
-        handle
-    }
-
     fn is_scaling_factor_cached(&self, decimals: usize) -> bool {
```

Comment:
I think this will be confusing for future developers. We should make this just a getter, and call the setter explicitly from ManagedDecimal.

Performance contraints don't seemt to justify it